WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -362,22 +365,48 @@ class AztecPostViewController: UIViewController {
 
 // MARK: - Actions
 extension AztecPostViewController {
-    @IBAction func cancelEditingAction() {
+
+    @IBAction func closeWasPressed() {
         cancelEditing()
     }
 
-    @IBAction func displayBlogSelector() {
+    @IBAction func blogPickerWasPressed() {
+        guard isSingleSiteMode == false else {
```

Comment:
@diegoreymendez thanks for the feedback!. I'll file another issue to track the change in behavior, i've just focused, as of now, in replicating the Hybrid's Editor behavior.

Thanks!!